src/olmo_core/data/numpy_dataset.py

```diff
@@ -1364,6 +1485,14 @@ class NumpyDatasetConfig(Config):
     """
     The type of dataset.
     """
+    bust_index_cache: bool = False
+    """
+    Whether or not to bust the index cache.
+    """
```

Comment:
It looks like this is only used in the `NumpyFSLDatasetMixture`? Is it still needed or was it just for debugging? Overwriting the cache files could create a race condition if they're in-use by another training process, so I'd prefer to remove this option.

Reply:
Yeah this was really just for debugging, the race condition is reason enough to drop it.